### ISSUE TITLE
Microsoft Excel: No Information Reported when Report Tables Setting Enabled and report coordinates disabled

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5395]
**Reported by elliott94 on 2015-09-30 06:53**
STR:
1. Enable the "Report Tables" setting from within NVDA.
2. Open a new Excel workbook, and navigate around the visible cells.

Expected: NVDA should announce "cell", as is the case when the reporting of tables is disabled.

Actual: No feedback is given, which may be confusing to some users.

**Comment by leonardder** [comment]
Honestly, I don't get why this goes wrong. Isn't nVDA supposed to read the role if not name?

**Comment by elliott94** [comment]
Unfortunately I don't have a copy of Excel that I can use to test this right now; if it's not too much trouble, would anybody else mind testing this?

**Comment by Adriani90** [comment]
Ok, I will close this as works for me since according to my tests the issue cannot be reproduced. @elliott94 feel free to comment if the issue is occuring again and we can reopen it. Thanks for reporting.
